.github/gh_matrix_builder.py

```diff
@@ -66,6 +66,7 @@ def build_release_config(overrides):
     "build_type": "Release",
     "pg_build_args": "",
     "tsdb_build_args": "-DWARNINGS_AS_ERRORS=ON -DREQUIRE_ALL_TESTS=ON",
+    "installcheck_args": "IGNORES='bgw_db_scheduler bgw_db_scheduler_fixed chunk_merge'",
```

Comment:
Why does chunk_merge not work in a release configuration? I think this should not be part of the PR and investigated why it doesnt work.